### PR TITLE
Functional tests for camera

### PR DESCRIPTION
**Pull request by Rboin**
Also fixed the build error

### CODE REVIEW

File: docs/test_document/chapters/functionalities/camera.tex [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+Robin Koning & June, 6, 2017                               		 
+\\\hline
+\end{tabularx}
+
+\begin{tabularx}{\textwidth}{|X|X|}
+\hline
+\rowcolor{lightgray}\textcolor{white}{\textbf{Test scenario}} &
+\textcolor{white}{\textbf{Desired result}}       
+\\\hline
+Camera is near the right edge and camera position is \{worldsize.x,0\}. &
```
Comment (JeroenVinke) [inline docs/test_document/chapters/functionalities/camera.tex]: Should this be "Cursor is near the right edge" ? Similar in other tests

Comment (Rboin) [inline docs/test_document/chapters/functionalities/camera.tex]: The other test is just moving the camera, without being at the edge of the world.

Comment (JeroenVinke) [inline docs/test_document/chapters/functionalities/camera.tex]: I understand, but what I mean is that you're not saying anything about the cursor, and the camera only moves when the cursor is in a particular position. So shouldn't the test be "**Cursor** is near the right edge and camera position is {worldsize.x,0}"?

Comment (Rboin) [inline docs/test_document/chapters/functionalities/camera.tex]: Ah I see, yes that would be correct.

**Comment by JeroenVinke** [comment]
Good to go!

**Pull request merged**